__test__/index.test.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-env jest */
+
+import responseToDataURI from '../src/index'
```

Comment:
I renamed data URI to URL everywhere; please update all occurrences of 'uri' in this file too.